commands.js

```diff
@@ -64,6 +64,28 @@ class Commands {
     selfDestructMessage(() => message.reply('added time to user.'))
   }
 
+  async bitrate (message) {
```

Comment:
You'll need to add an entry to the table at the bottom of the file to expose this command.